provider/generic_repository.go

```diff
@@ -47,8 +48,16 @@ func (p *GenericProvider) Gather(user string) ([]GitRepository, error) {
 		return nil, errors.New("This provider requires a object. Example: src-fingerprint -p repository -u 'git@github.com:GitGuardian/gg-shield.git'") // nolint
```

Comment:
I think we should rather user : "https://github.com/GitGuardian/src-fingerprint"...